lib/ts/recipe/passwordless/components/themes/signInUp/emailForm.tsx

```diff
@@ -65,6 +60,10 @@ export const EmailForm = withOverride(
                         userContext: {},
                     });
 
+                    if (response.status === "GENERAL_ERROR") {
+                        throw new STGeneralError(response.message);
+                    }
```

Comment:
Wouldn't the `STGeneralError` be thrown by `props.recipeImplementation.createCode` in case it's a GENERAL_ERROR response?

Reply:
But passwordless hasn't been implemented in web-js yet. So you will need this for now. Resolving this.